AppCenterDataStorage/AppCenterDataStorage/Internal/LocalStore/MSDBDocumentStore.m

```diff
@@ -65,14 +64,14 @@ - (BOOL)createUserStorageWithAccountId:(NSString *)accountId {
 - (BOOL)upsertWithToken:(MSTokenResult *)token
         documentWrapper:(MSDocumentWrapper *)documentWrapper
               operation:(NSString *_Nullable)operation
-                options:(MSBaseOptions *)options {
+       deviceTimeToLive:(NSInteger)deviceTimeToLive {
   // Compute expiration time as now + device time to live (in seconds).
   // If device time to live is set to infinite, set expiration time as null in the database.
   // Note: If the cache/store is meant to be disabled, this method should not even be called.
   NSTimeInterval now = NSDate.timeIntervalSinceReferenceDate + NSTimeIntervalSince1970;
```

Comment:
Question: why not '[[NSDate date] timeIntervalSince1970]'? It would look easier to understand as for me.

Reply:
Someone added a comment about that apparently. It's saving some memory (one less object to allocate).